docs/operate/get-started/supported-hardware.md

```diff
@@ -5,5 +5,367 @@ weight: 20
 layout: "docs"
 type: "docs"
 no_list: true
-description: "TODO"
+description: "Use supported hardware with your machine."
+modulescript: true
 ---
+
+Viam supports a wide variety of sensors, cameras, motors, robotic arms, and other physical hardware.
+Viam also supports various software services such as [data capture](/data-ai/get-started/capture-sync/), [computer vision](/data-ai/ai/create-dataset/), and [motion planning](/operate/mobility/move-arm/), designed to integrate seamlessly with the hardware driver modules.
+
+Any hardware that is not already supported by a Viam {{< glossary_tooltip term_id="module" text="module" >}} can be added into Viam's system of modular resources by [creating a new module](../other-hardware/) that provides a driver for the hardware.
+You can also create a module to support new software-only functionality.
+
+In addition to physical hardware, there are "virtual" hardware modules that do not directly drive any physical hardware, but rather augment physical hardware with another layer of abstraction, or add functionality that has little to do with hardware, for example:
```

Comment:
best to try to describe what something is rather than what it isn't. I think ok here to leave that to the examples though
```suggestion
In addition to physical hardware, there are "virtual" hardware modules that do not directly drive any physical hardware, but rather augment physical hardware with another layer of abstraction, or add other functionality, for example:
```